packages/@textlint/kernel/test/rule-fixer/rule-fixer-test.ts

```diff
@@ -1,6 +1,6 @@
 // LICENSE : MIT
 "use strict";
-import { TxtNode } from "../../src/textlint-kernel-interface";
+import { TxtNode } from "@textlint/ast-node-types";
```

Comment:
nice catch!
 
I want to find this pattern automatically :)
https://github.com/textlint/textlint/issues/373